patches/chrome-test-chromedriver-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/test/chromedriver/BUILD.gn b/chrome/test/chromedriver/BUILD.gn
+index ab46b4e2de94f567e2f010f6badae325db24ac61..0929168b9f17b4aa2bdd6e721bf71af8909d90d0 100644
+--- a/chrome/test/chromedriver/BUILD.gn
++++ b/chrome/test/chromedriver/BUILD.gn
+@@ -304,7 +304,7 @@ source_set("lib") {
+ }
+ 
+ executable("chromedriver") {
+-  testonly = true
++#  testonly = true
```

Comment:
we should not be doing this. What is the error without these changes?

Reply:
<pre>
ERROR at //brave/BUILD.gn:264:3: Test-only dependency not allowed.                                                                                                                              copy("brave_chromedriver_dist_resources") {                                                                                                                                                   ^------------------------------------------                                                                                                                                                 //brave:brave_chromedriver_dist_resources                                                                                                                                                     which is NOT marked testonly can't depend on                                                                                                                                                  //chrome/test/chromedriver:chromedriver                                                                                                                                                       which is marked testonly. Only targets with "testonly = true"                                                                                                                                 can depend on other test-only targets.
</pre>

Reply:
https://github.com/brave/brave-core/pull/2855/commits/c18e0bd4844d1efcb60768b6159669011db920a5